mlflow/tracking/fluent.py

```diff
@@ -1329,19 +1333,30 @@ def print_auto_logged_info(r):
         "pytorch_lightning": pytorch.autolog,
     }
 
-    def get_autologging_params(autolog_fn):
+    def get_autologging_params(integration, autolog_fn):
         try:
             needed_params = list(inspect.signature(autolog_fn).parameters.keys())
-            return {k: v for k, v in locals_copy if k in needed_params}
+            param_spec = inspect.signature(autolog_fn).parameters
+            default_params = {param.name: param.default for param in param_spec.values()}
+            return {k: AutologgingConfigManager.get_config(integration, k, default_params[k]) for k in needed_params}
```

Comment:
Previously, this function gathered the values for params required by the autolog fn from `locals()`. We no longer want to directly call autolog functions with the values passed into `mlflow.autolog`.
Changed this function to return the correct configs from `ConfigManager` instead.